### ISSUE TITLE
Confuse about Implementation of GeneralModelResnet (models.py)

### ISSUE THREAD

**Issue by Sabo30** [RuBP17/AlphaDou#7]
Hi, thanks your great works. I read through your paper and your code. I confused about this line of code:
out = torch.cat([x, x, x, x, out], dim=-1)

Could you please explain why you need to duplicate input 'x' 4 times? 
Many thanks

**Comment by RuBP17** [comment]
the values of `x` will directly influence the final prediction (the bid info, the number of bombs), so we place them in full-connection layers instead of conv layers. However, compare to the `out`, the length of `x` (18) is too short (`out` : 2016), we are concerned that the network may neglect the `x`, so we repeat it 4 times to emphasize the `x`. Although we haven't tested other repetition counts, repeating it 4 times yielded good results. Similarly, Douzero ResNet also employs this '4 times' approach, which we have adopted.
